adding missing docs (#61)
percona-platform/portal-doc: docs/advisors/checks/innodb-strict-mode.md

# MySQL InnoDB strict mode not correct

## Description

> When InnoDB strict mode is disabled, it's possible to have incompatible settings for 

Author: @catalinaadam ok for me

> When InnoDB strict mode is disabled, it's possible to have incompatible settings for 

Reviewer: changed

KEY_BLOCK_SIZE, ROW_FORMAT, DATA DIRECTORY, TEMPORARY, and TABLESPACE table options.  Enabling it will force InnoDB to make sure these create options are compatible with each other and other settings.

Also, when enabled, a row size check is done when creating or altering a table, to prevent INSERT or UPDATE from failing due to the record being too large for the selected page size.

## Rule
```
        SELECT @@global.innodb_strict_mode;
```

## Resolution

> This setting (innodb_strict_mode) is enabled by default, and we strongly recommend leaving it enabled.  This is an on-line change that can be applied with:

Reviewer 2: ```suggestion
The **innodb_strict_mode** setting is enabled by default. Percona strongly recommends leaving this setting enabled.  
This is an online change that you can apply with:
```
